src/EnergyPlus/Coils/CoilCoolingDXCurveFitSpeed.cc

```diff
@@ -349,16 +350,27 @@ void CoilCoolingDXCurveFitSpeed::size()
     SizingString = "Gross Sensible Heat Ratio";
     DataSizing::DataFlowUsedForSizing = this->evap_air_flow_rate;
     DataSizing::DataCapacityUsedForSizing = this->rated_total_capacity;
-    ReportSizingManager::RequestSizing(CompType, CompName, SizingMethod, SizingString, this->grossRatedSHR, PrintFlag, RoutineName);
+    if (this->grossRatedSHR == DataSizing::AutoSize && this->parentOperatingMode == 2) {
```

Comment:
I would like to try refactoring some here to avoid the speed having to know this much about the parent mode.  But that's for another day.